src/widgets/histogram/content-view.js

```diff
@@ -76,7 +76,6 @@ module.exports = cdb.core.View.extend({
     if (data && data.length > 0) {
```

Comment:
@xavijam Would you remove this whole storeBounds method for me? I should have removed it in a previous PR. It can be :fire::fire::fire: